app/assets/styles/global/_pages.scss

```diff
@@ -185,3 +185,16 @@ main {
     float: right;
   }
 }
+
+.staging-footer {
```

Comment:
I think this is our only example of a sticky banner. What do you think of breaking out the styles into 
- sticky-banner
- staging-footer
So we can reuse this later for the new emergencies banner (or any other feature that might come along)

Reply:
Check: 4053d2e